docs/GraFx-Studio/concepts/conversion/index.md

```diff
@@ -0,0 +1,37 @@
+# Conversion of Desktop files
+
+## Intro
+
+Imagine the scenario where you have a design created in a Desktop application. This file should be the basis for your Smart Template.
+
+AN easy design could be recreated in GraFx Studio, for a more complex file it would be handy to have a conversion from your existing file to GraFx Studio.
+
+That is wat a conversion does.
+
+## Intermediate File Format
+
+To accomodate several application, we designed the process to enable Desktop applications to export to an Intermediate File Format.
+
+Once in GraFx Studio, you can import the file you (or a plugin) created into GraFx Studio.
+
+## Process
+
+### Export with plugin
+
+For these applications, we created plugins to export to the Intermediate File Format.
+
+The plugin will create a zip file, with the Intermediate File, and all relevant assets.
+
+### Import on CHILI GraFx
+
+Once in GraFx Studio, you can import the exported (zip) file.
+
+![screenshot](plugin1.png)
+
+## Limitations
```

Comment:
WARNING & ERRORS
Warning

- Convert to pdf = not editable in Studio after conversion
- ‘Ignore’ = editable in Studio after conversion (but object is exported with default property: dotted line → solide line)

Error = create a placeholder frame because error blocks conversion (examples: font missing, link missing embedded links & images)

Reply:
> Error = create a placeholder frame because error blocks conversion (examples: font missing, link missing embedded links & images)

Plugin supports only 2 options for items with unsupported features:
- export as vector graphics (.pdf format)
- 'ignore' (**will export item** ommiting unsupported feature)
- we do not block conversion and do not create 'frame placeholders'